webui/react/src/pages/ExperimentDetails.module.scss

```diff
@@ -1,5 +0,0 @@
-.base {
-  .topRow {
-    height: 40rem;
```

Comment:
@hkang1 I removed those because were preventing the tab content to be scrolled on mobile.